assets/dev/js/editor/document/atomic-widgets/commands/styles.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * @typedef {import('../../../container/container')} Container
+ */
+export class Styles extends $e.modules.editor.document.CommandHistoryDebounceBase {
+	/**
+	 * Function getSubTitle().
+	 *
+	 * Get sub title by container.
+	 *
+	 * @return {string} sub title
+	 */
+	static getSubTitle() {
+		return 'Styles';
+	}
+
+	/**
+	 * Function restore().
+	 *
+	 * Redo/Restore.
+	 *
+	 * @param {{}}      historyItem
+	 * @param {boolean} isRedo
+	 */
+	static restore( historyItem, isRedo ) {
+		const data = historyItem.get( 'data' );
+
+		historyItem.get( 'containers' ).forEach( ( /* Container */ container ) => {
+			const changes = data.changes[ container.id ];
+
+			$e.run( 'document/atomic-widgets/styles', {
```

Comment:
This would cause an error if reverted to a widget with no styles (as all required props in `args` will be missing, i.e. `meta`, `props` and `bind`)

Also - it should detect if the style is missing in `changes.old` and call the `set-settings` command with the responsive `bind` property without the removed `styleDefId`

Reply:
done, as we talked about.